### DISCUSSION TITLE
docs/s3-backup/

### DISCUSSION THREAD

**Discussion by giscus** [ricsanfre/pi-cluster#212] [Announcements]
# docs/s3-backup/

How to deploy a Minio S3 object storage server in Bare-metal environment as backup backend for our Raspberry Pi Kubernetes Cluster.

https://picluster.ricsanfre.com/docs/s3-backup/

**Comment by i5Js** [comment]
There is a typo in the schedule template: included_resources doesn't work, it is includedResources

**Reply by ricsanfre** [reply]
Thanks for the heads-up @i5Js. Typo already fixed

**Comment by ilbarone87** [comment]
MinIO going to remove majority of their features and console UI from latest version. Do not upgrade!!
https://www.reddit.com/r/selfhosted/comments/1kva3pw/avoid_minio_developers_introduce_trojan_horse/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

https://github.com/minio/object-browser/pull/3509

If anyone has a good replacement it's time to shout. Redditors are ssuggesting GarageS3 but lacks of UI and some features.

**Comment by ilbarone87** [comment]
@ricsanfre given the current situation with MinIO and their awful rug pull move, are you planning to move away from minio? I have had a look around already but doesn't seem there is a product mature as minio was.
This is the last move https://github.com/minio/minio/issues/21647

**Reply by ricsanfre** [reply]
Hi @ilbarone87, yes I have plans to move away from MinIO the same way I moved away from Linkerd or Bitnami Helm charts when the companies behind them did similar changes in their opensource strategy.

I did not start a deep analysis yet but as you said the main problem is the maturity of the alternatives. Have you tried any of the alternatives like [Garage](https://garagehq.deuxfleurs.fr/), [RustFS](https://github.com/rustfs/rustfs) or [SeaweedFS](https://github.com/seaweedfs/seaweedfs)?

**Reply by ilbarone87** [reply]
Indeed these 2 are the on the shortlist, but haven't started on any of them yet. I'm trying to delay the move as much as I can as minIO offers still more features than them (like UI or built-in SSO). Hoping that the miniIO situation will give a push to the developing of at least one of them.
